packages/apps/tools/src/constants/kadena.ts

```diff
@@ -34,13 +34,6 @@ export const kadenaConstants: KadenaConstants = {
     estatsHost: () =>
       env('KADENA_TESTNET_ESTATS', 'estats.testnet.chainweb.com'),
   },
-  testnet05: {
-    label: 'Testnet05',
-    API: env('KADENA_TESTNET05_API', 'api.testnet05.chainweb.com'),
-    apiHost: ({ networkId, chainId }) =>
-      `https://${kadenaConstants.testnet05.API}/chainweb/0.0/${networkId}/chain/${chainId}/pact`,
-    estatsHost: () => '',
-  },
```

Comment:
Let's separate `kadenaConstants` from `networks. 

Ie: have two exports `networks` and `constants`.
Also include a separate one called `networkNames` which is just the network IDs

```typescript
export const networks: { [K in DefinedNetwork]: {
    label: string;
    API: string;
    apiHost: (params: { networkId: string; chainId: string }) => string;
    estatsHost: () => string;
  } } =  { /* ... */ };
export const networkIds = Object.keys(networks);
```